<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePort">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labPort">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreateRoom">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>创建聊天室</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
